cpp-examples/demos/crypto.cc

```diff
@@ -0,0 +1,184 @@
+/*
```

Comment:
Is there anything different about crypto.cc (vs demo3.cc or whatever) aside from the connection string `localhost:10042` ?

Reply:
I'm not asking you to do anything, but if you want me to take a pass over this and consolidate these demos a little more I can

Reply:
I'm moving that file out of this PR and onto a different one, we can discuss separately when we (just you and me and not the rest of the ppl in this review) discuss the crypto demo part for tuco y tico etc.  Thanks for the comments btw, I made the changes you suggested earlier.

Reply:
SGTM